reference/dic_tags.rst

```diff
@@ -260,6 +260,10 @@ Add this tag to a service and its class won't be preloaded when using
             ->addTag('container.no_preload')
         ;
 
+If some service tagged with ``container.no_preload`` is added as an argument of
+other service, the ``container.no_preload`` tag is applied automatically to that
+service too.
```

Comment:
```suggestion
another service, the ``container.no_preload`` tag is applied automatically to that
service too since Symfony 5.1.
```

Reply:
Thanks! the "since Sf 5.1" part is not needed because the entire "no_preload" feature was introduced in 5.1 and we already have the "versionadded" directive at the beginning of this section. But I'll do the other change while merging.